src/dialogs/preferences/AnalOptionsWidget.cpp

```diff
@@ -33,7 +45,13 @@ AnalOptionsWidget::AnalOptionsWidget(PreferencesDialog *dialog)
 
     ui->analyzePushButton->setToolTip("Analyze the program using radare2's \"aaa\" command");
     auto *mainWindow = new MainWindow(this);
-    connect(ui->analyzePushButton, &QPushButton::clicked, mainWindow, &MainWindow::on_actionAnalyze_triggered);
+    connect(ui->analyzePushButton, &QPushButton::clicked, mainWindow,
+            &MainWindow::on_actionAnalyze_triggered);
+    connect(ui->analInComboBox, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged),
+            this, &AnalOptionsWidget::updateAnalIn);
+    connect(ui->ptrDepthSpinBox, static_cast<void (QSpinBox::*)(int)>(&QSpinBox::valueChanged), this,
```

Comment:
For overloaded signals you can do this, which is slightly less verbose.
```suggestion
    connect<void (QSpinBox::*)(int)>(ui->ptrDepthSpinBox, &QSpinBox::valueChanged, this,
```

Reply:
Thanks, that is indeed somewhat nicer to read. I changed both the `ptrDepthSpinBox` and the `analInComboBox`. Thanks!